packages/office-ui-fabric-react/src/components/Button/IconButton/IconButton.styles.ts

```diff
@@ -26,11 +26,17 @@ export const getStyles = memoizeFunction((
     },
 
     rootHovered: {
-      color: theme.palette.themeDarker
+      color: theme.palette.themeDarker,
+      backgroundColor: theme.palette.neutralLighter,
```

Comment:
Are there semantic color slots we can use for these?

Reply:
Yeah.
`semanticColors.buttonBackground` = `palette.neutralLighter` (the color Ben requested for **Hovered** state)
`semanticColors.buttonBackgroundHovered` = `palette.neutralLight` (the color Ben requested for **Pressed** state)

Reply:
Awesome--if that's the case, I'd think we should use those instead so IconButton can pick up themes moving forward. @betrue-final-final @phkuo for FYI

Reply:
Alright, I'll change all those over. Is there a `semanticColor` for `palette.themePrimary` or leave those alone in this case?

Reply:
@Jahnp One more review?

Reply:
There are, though I'm not sure offhand which would be best. `inputBackgroundChecked` is the closest conceptually. You might take a look at the mappings in [`_makeSemanticColorsFromPalette`](https://github.com/OfficeDev/office-ui-fabric-react/blob/ebe601051c081838ae9d029e1020f316258d148a/packages/styling/src/styles/theme.ts#L120) for reference.

Reply:
I'm hesitant to use a semanticColor intended for the background on a foreground element.

Reply:
Totally fair--duh. I must have missed the background portion. There doesn't seem to a perfect match--[`menuIcon`](https://github.com/OfficeDev/office-ui-fabric-react/blob/ebe601051c081838ae9d029e1020f316258d148a/packages/styling/src/styles/theme.ts#L165) seems like the next closest, but it feels wrong to apply "menu" to something that isn't in a menu.

@phkuo -- is this a case where we'd want to consider a new slot, or is it acceptable to stretch the usage of an existing slot, like using `menuIcon` for the foreground of an icon button?

Reply:
There are three possible ones I'd use: `menuIcon`, `menuHeader`,  or `link`. 
@mikewheaton Does `palette.themePrimary` need to be replaced in scenarios like this? Doesn't it get its value from the theme HOC too?

Reply:
@jordandrako both palette and semantic colors theme. If uncustomized, semantic colors will pick up an appropriate palette color.

Don't try to "stretch" the meaning of a semantic slot too far, we shouldn't use menu slots for buttons. We do need new color slots for "text buttons", buttons that aren't outlined.

Reply:
Alright, then I think we should leave the themePrimary uses alone since they're applied to the IconButton's foreground color. @Jahnp cool?

Reply:
Ok, as long as having a mix of semantic & palette colors is fine, that works for me.